src/test/groovy/net/ossindex/integrationtests/OssIndexAuditPluginTests.groovy

```diff
@@ -163,8 +163,66 @@ class OssIndexAuditPluginTests extends Specification {
         result.task(":audit").outcome.is(SUCCESS)
     }
 
-    def "a project which ignores a specific artifact but has another vulnerability should fail"() {
-        given: "A project with 2 vulnerable artifacts and one ignore"
+//    def "a project which ignores a specific artifact but has another vulnerability should fail"() {

```

Comment:
I am assuming we can delete these comments now? Or should this be added back in?